spec/lib/debt_management_center/statement_identifier_service_spec.rb

```diff
@@ -7,6 +7,194 @@
                type: :service do
   describe '#get_mpi_data' do
     Flipper.disable(:va_v3_contact_information_service)
+
+    context 'given edipi statement' do
+      edipi = '492031291'
```

Comment:
I would make this a `let` instead of a direct variable definition

Reply:
I just copied the previous code and enabled the service. I can make a ticket to address this during the refactor? This spec will be changed when the new service replaces the old service and I agree, there are better ways to go about this.